bitnami/node-exporter/values.yaml

```diff
@@ -0,0 +1,222 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+  labels: {}
+    # foo: bar
+
+## String to partially override node-exporter.fullname template (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override node-exporter.fullname template
+##
+# fullnameOverride:
+
+## Role Based Access
+## Ref: https://kubernetes.io/docs/admin/authorization/rbac/
+##
+rbac:
+  create: true
+
+  ## RBAC API version
+  ##
+  apiVersion: v1beta1
+
+  ## Podsecuritypolicy
+  ##
+  pspEnabled: true
+
+## Bitnami Node Exporter image version
+## ref: https://hub.docker.com/r/bitnami/node-exporter/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/node-exporter
+  tag: 0.18.1-debian-9-r172
+
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+## Additional command line arguments to pass to node-exporter
+extraArgs: {}
+  # collector.filesystem.ignored-mount-points: "^/(dev|proc|sys|var/lib/docker/.+)($|/)"
+  # collector.filesystem.ignored-fs-types: "^(autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$"
```

Comment:
Does `yamllint` complain about the format of these comments? It's likely you find the error below:
```
warning  comment not indented like content  (comments-indentation)
```